pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -325,14 +325,14 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 				err)
 			dswp.desiredStateOfWorld.AddErrorToPod(uniquePodName, err.Error())
 			allVolumesAdded = false
+		} else {
+			klog.V(4).Infof(
+				"Added volume %q (volSpec=%q) for pod %q to desired state.",
+				podVolume.Name,
+				volumeSpec.Name(),
+				uniquePodName)
 		}
 
-		klog.V(4).Infof(
-			"Added volume %q (volSpec=%q) for pod %q to desired state.",
-			podVolume.Name,
-			volumeSpec.Name(),
-			uniquePodName)
-
 		if utilfeature.DefaultFeatureGate.Enabled(features.ExpandInUsePersistentVolumes) {
```

Comment:
@gnufied do we actually want to check fs resize if volume was not added to dsw?

Reply:
I have made one more change by lifting feature gate detection out of the loop (stored in a variable).
We just check the variable inside the loop.

Reply:
@gnufied
Kindly provide your feedback.

Reply:
@gnufied 
Please review.

Reply:
We can follow up on this later